mlir/test/Dialect/Arith/canonicalize.mlir

```diff
@@ -985,6 +985,16 @@ func.func @tripleMulIMulII32(%arg0: i32) -> i32 {
   return %mul2 : i32
 }
 
+// CHECK-LABEL: @tripleMulLargeInt
+//       CHECK:   return
+func.func @tripleMulLargeInt(%arg0: i256) -> i256 {
+  %0 = arith.constant 3618502788666131213697322783095070105623107215331596699973092056135872020481 : i256
+  %c5 = arith.constant 5 : i256
```

Comment:
Could you also check the result of the fold is correct? This is useful and missing test-coverage given that it used to assert previously and definitely worth adding 🙂